timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -351,19 +350,17 @@ void declare_context(py::module &m) {
             "multiple_steps",
             [](Context &ctxt, const int n_steps, int store_x_interval) -> py::tuple {
                 // (ytz): I hate C++
-                int x_interval = (store_x_interval <= 0) ? n_steps : store_x_interval;
-                std::array<std::vector<double>, 2> result = ctxt.multiple_steps(n_steps, x_interval);
-
                 int N = ctxt.num_atoms();
                 int D = 3;
-                int F = result[0].size() / (N * D);
-                py::array_t<double, py::array::c_style> out_x_buffer({F, N, D});
-                std::memcpy(out_x_buffer.mutable_data(), result[0].data(), result[0].size() * sizeof(double));
-
-                py::array_t<double, py::array::c_style> box_buffer({F, D, D});
-                std::memcpy(box_buffer.mutable_data(), result[1].data(), result[1].size() * sizeof(double));
 
-                return py::make_tuple(out_x_buffer, box_buffer);
+                int x_interval = (store_x_interval <= 0) ? n_steps : store_x_interval;
+                int n_samples = store_x_interval > n_steps ? 0 : n_steps / x_interval;
```

Comment:
is this needed?

if `store_x_interval > n_steps` doesn't `n_steps / x_interval` in int-math always result in 0?

Reply:
(either way, should probably add a doc-string clarifying intent of the `x_interval` vs `store_x_interval`)

Reply:
Added docstring in https://github.com/proteneer/timemachine/pull/1370/commits/aecc151a8f5799caeb03508af0e57f888350ca83